tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/conversion/MavenConversionTest.java

```diff
@@ -263,4 +267,16 @@ private void finishConversionDialog(){
 		}
 		return libs;
 	}
+	
+	public void replaceInPom(String target, String replacement, boolean save) {
```

Comment:
Where is this used?

Reply:
Well, the PR has been edited several times due to some resolved issue and due to newly appeared issues. I must have created/copied this method and I must have forgotten to delete it when I come up to the conclusion that I do not need it. Deleted.